api/app/signals/apps/questionnaires/models/utils.py

```diff
@@ -0,0 +1,91 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 Gemeente Amsterdam
+import os
+
+import matplotlib.pyplot as plt
+import networkx
+from django.utils import timezone
+from networkx.drawing.nx_pydot import graphviz_layout
+
+import pydot  # noqa
+
+
+def make_nx_graph(question_graph):
+    nx_graph = networkx.DiGraph()
```

Comment:
I have a strong suspicion that we need a `MultiDiGraph` here:

A node with `n` outgoing edges for which we set a `Choice` (~ predefined answers) will not have a default option. If we want a default as well we have to allow a second edge with `choice=None` to be defined for that default. That would turn our directed graph of questions in which we allow parallel edges (and hence a `MultiDiGraph` in the NetworkX world).

Side note: if we really want to avoid parallel edges, we could consider a `default` Boolean field on the Edge model.

Reply:
I just copied this from the code that is already in the master branch. If we really want to use a `MultiDiGraph` then we can add it in the next PR/Refactor. I think this is something you are already working on in PR #861 https://github.com/Amsterdam/signals/pull/861/files#diff-10364d2fe9ac0385fe9a77a5e47dd72c73d9286d9037940b6a209f330ab73aceR52